src/pods/toolbar/components/export-button/export-button.business.ts

```diff
@@ -28,6 +28,78 @@ export const calculateTableEndYPosition = (table: TableVm): number => {
   return table.y + tableHeight;
 };
 
+export const isTableOverlap = (table: TableVm, tables: TableVm[]): boolean => {
+  if (tables.length === 0) {
+    return false;
+  }
+
+  const expandedFields = expandTableFields(table.fields);
+  const tableEndXPosition = table.x + TABLE_CONST.TABLE_WIDTH;
+  const tableEndYPosition = calculateTableEndYPosition({
+    ...table,
+    fields: expandedFields,
+  });
+
+  return tables.some(t => {
+    if (t.id === table.id) {
+      return false;
+    }
+
+    const tExpandedFields = expandTableFields(t.fields);
+    const tEndXPosition = t.x + TABLE_CONST.TABLE_WIDTH;
+    const tEndYPosition = calculateTableEndYPosition({
+      ...t,
+      fields: tExpandedFields,
+    });
```

Comment:
Encapsulate all this in a method, signature proposal:

```ts
const tableBBottomRightPosition =  calculateTableBottomRightPosition(tableB);
```

Reply:
i think it's not correct this name because you dont calculate the bottom right side position in the table, this method calculates de end Y position but you are at the beginning. I have used calculateTableButtonCoordinate.

done